src/core/client/admin/components/MediaContainer/ExternalMedia.tsx

```diff
@@ -9,15 +9,26 @@ interface Props {
 
 const ExternalMedia: FunctionComponent<Props> = ({ url, siteID }) => {
   const cleanUrl = encodeURIComponent(url);
+
+  const attrs: HTMLProps<HTMLIFrameElement> = {};
+
+  // Force loading=lazy to be added for enhanced loading support on supported
+  // browsers.
+  (attrs as any).loading = "lazy";
```

Comment:
I know you wanted to avoid `any` if at all possible. If you have time, cast it to a typed interface and set it?